### PR TITLE
chore: Refactor hub for 3.0.0

### PR DESCRIPTION
**Pull request by neelanjan00**
Refactor hub for 3.0.0

### CODE REVIEW

**Comment by amityt** [comment]
@neelanjan00  can you remove charts and workflows dir, rest of the changes look good.

**Review by amityt — APPROVED** [review]

File: faults/aws/aws-ssm-chaos-by-id/aws-ssm-chaos-by-id.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - AWS
+  platforms:
+    - AWS
+  chaosType: infra
+  labels:
+    app.kubernetes.io/component: chartserviceversion
+    app.kubernetes.io/version: ci
+  links:
+    - name: Documentation
+      url: https://docs.harness.io/category/kl0mxwpfw1-hce-category
```
Comment (Jonsy13) [inline faults/aws/aws-ssm-chaos-by-id/aws-ssm-chaos-by-id.chartserviceversion.yaml]: The documentation links should be pointing to OSS docs

File: faults/aws/ebs-loss-by-tag/ebs-loss-by-tag.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  createdAt: 2022-07-28T10:28:08Z
+  name: ebs-loss-by-tag
+  version: 0.1.0
+  annotations:
+    categories: AWS
+    vendor: Harness
```
Comment (Jonsy13) [inline faults/aws/ebs-loss-by-tag/ebs-loss-by-tag.chartserviceversion.yaml]: Vendor will be Harness or LitmusChaos?

File: faults/aws/aws-ssm-chaos-by-id/fault.yaml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        verbs: ["get", "list", "create"]
+      # for configuring and monitor the experiment job by the chaos-runner pod
+      - apiGroups: ["batch"]
+        resources: ["jobs"]
+        verbs: ["create", "list", "get", "delete", "deletecollection"]
+      # for creation, status polling and deletion of litmus chaos resources used within a chaos workflow
+      - apiGroups: ["litmuschaos.io"]
+        resources: ["chaosengines", "chaosexperiments", "chaosresults"]
+        verbs: ["create", "list", "get", "patch", "update", "delete"]
+    image: "chaosnative/go-runner:ci"
```
Comment (Jonsy13) [inline faults/aws/aws-ssm-chaos-by-id/fault.yaml]:
```suggestion
    image: "litmuschaos/go-runner:ci"
```
Please change everywhere for `go-runner`

File: faults/kubernetes/kubernetes.package.yaml [hunk 4]
```diff
@@ -75,15 +79,24 @@ experiments:
...
-  - name: pod-http-status-code
-    CSV: pod-http-status-code.chartserviceversion.yaml
-    desc: "pod-http-status-code"
+  - name: node-network-latency
+    CSV: node-network-latency.chartserviceversion.yaml
+    desc: "node-network-latency"
+  - name: node-network-loss
+    CSV: node-network-loss.chartserviceversion.yaml
+    desc: "node-network-loss"
+  - name: time-chaos
```
Comment (Jonsy13) [inline faults/kubernetes/kubernetes.package.yaml]: Don't think we have timechaos in OSS?

**Comment by Jonsy13** [comment]
The build is failing, @ispeakc0de Can you please check?

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**